contracts/RestrictedNORI.sol

```diff
@@ -464,7 +463,10 @@ contract RestrictedNORI is
     address account
   ) public view returns (uint256) {
     Schedule storage schedule = _scheduleIdToScheduleStruct[scheduleId];
-    uint256 scheduleTrueTotal = _scheduleTrueTotal(schedule, scheduleId);
+    uint256 scheduleTrueTotal = _scheduleTrueTotal(
+      schedule.totalClaimedAmount,
+      scheduleId
+    );
     uint256 claimableForAccount;
     // avoid division by 0
     if (scheduleTrueTotal == 0) {
```

Comment:
should this also avoid division _**of**_ zero? like we added [here](https://github.com/nori-dot-eco/contracts/pull/249/files/998a7c2f78de2bf6a1f41f2c5161c04e02449ab5..fb97bb8a727a24cdc034f908b27899c6a7b61e26#diff-de34eb981fc0fa6e6b114737c58140e5e1b03e6f25139d619de635e4e0a9d7e0R989)? Any other places we need to add this?

Reply:
will check for all comments like that